scripts/install.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+cd `dirname $0`/..
+
+if [[ "$(node --version)" < "v5" ]]; then
```

Comment:
Well, node v4 doesn't work for @kwalrath, and more often than not, forgetting to switch to v5 for Kathy has cause problems and results in wasted time.

Are you aware of anyone who _must_ stick with node 4 and who is required to build ng.io docs? (Kathy has to do this every time she tests out a PR for merging :) If not, IMHO, we should just just drop v4.